pkg/virt-handler/cmd-client/client.go

```diff
@@ -193,20 +191,20 @@ func SocketMonitoringEnabled(socket string) bool {
 }
 
 func IsSocketUnresponsive(socket string) bool {
+	exists, _ := diskutils.FileExists(socket)
+	// if the socket file doesn't exist, it's definitely unresponsive
+	if !exists {
```

Comment:
Should be able to func test this.  I think restarting the handler with a running VMI should do it: https://github.com/kubevirt/kubevirt/blob/main/tests/vmi_lifecycle_test.go#L500